config/manager/image_config.yaml

```diff
@@ -3,7 +3,7 @@ kind: ImageConfig
 metadata:
   name: image-config
 images:
-  system-logger: "k8ssandra/system-logger:latest"
+  system-logger: "k8ssandra/system-logger:v1.22.0-dev.e8bef6b-20240716"
```

Comment:
Will this tag change before release?

Reply:
Yeah, this tag is automatically modified in the release process.

Reply:
Awesome, this just surprised me.

Reply:
It's not intended to be in the PR, but it makes no difference as all the operations modify these values automatically in cass-operator (make deploy, release scripts etc).

We don't use :latest as real deployable tag.